care/facility/migrations/0408_alter_dailyround_consciousness_level_and_more.py

```diff
@@ -0,0 +1,254 @@
+# Generated by Django 4.2.8 on 2024-01-20 10:02
+
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    DailyRound = apps.get_model("facility", "DailyRound")
+    daily_round_objects = DailyRound.objects.all()
+    objects = []
+    for obj in daily_round_objects:
+        if obj.consciousness_level == 0:
+            obj.consciousness_level = None
+        if obj.left_pupil_light_reaction == 0:
+            obj.left_pupil_light_reaction = None
+        if obj.right_pupil_light_reaction == 0:
+            obj.right_pupil_light_reaction = None
+        if obj.limb_response_upper_extremity_left == 0:
+            obj.limb_response_upper_extremity_left = None
+        if obj.limb_response_upper_extremity_right == 0:
+            obj.limb_response_upper_extremity_right = None
+        if obj.limb_response_lower_extremity_left == 0:
+            obj.limb_response_lower_extremity_left = None
+        if obj.limb_response_lower_extremity_right == 0:
+            obj.limb_response_lower_extremity_right = None
+        if obj.rhythm == 0:
+            obj.rhythm = None
+        if obj.ventilator_mode == 0:
+            obj.ventilator_mode = None
+        if obj.ventilator_interface == 0:
+            obj.ventilator_interface = None
+        if obj.ventilator_oxygen_modality == 0:
+            obj.ventilator_oxygen_modality = None
+        if obj.insulin_intake_frequency == 0:
+            obj.insulin_intake_frequency = None
+        if obj.bp and obj.bp.get("systolic") and obj.bp.get("diastolic"):
+            if obj.bp.get("systolic") < 0 and obj.bp.get("diastolic") < 0:
+                obj.bp = {}
+            else:
+                if obj.bp.get("systolic") < 0:
+                    obj.bp.update({"systolic": None})
+                if obj.bp.get("diastolic") < 0:
+                    obj.bp.update({"diastolic": None})
```

Comment:
use individual queries for each field
https://docs.djangoproject.com/en/4.2/topics/db/queries/#querying-jsonfield

Reply:
Done